libs/utils/test.py

```diff
@@ -42,20 +42,31 @@ def _init(cls, conf, *args, **kwargs):
             cls.logger.setLevel(kwargs['loglevel'])
             kwargs.pop('loglevel')
 
-        cls.conf = conf
-
         cls._runExperiments()
 
     @classmethod
+    def _getTestConf(cls):
+        if not hasattr(cls, "test_conf"):
+            raise NotImplementedError("Please add `test_conf` attribute")
+        return cls.test_conf
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        if not hasattr(cls, "experiments_conf"):
+            raise NotImplementedError("Please add `experiments_conf` attribute")
+        return cls.experiments_conf
+
+    @classmethod
     def _runExperiments(cls):
         """
         Default experiments execution engine
         """
 
         cls.logger.info("%14s - Setup tests execution engine...", "LisaTest")
-        test_env = TestEnv()
+        test_env = TestEnv(test_conf=cls._getTestConf())
```

Comment:
Nope, we need the named parameter because we're using the default for `target_conf`.